offlineOnline.js

```diff
@@ -51,6 +51,20 @@ export const loadVersions = async (ref) => {
     return { versions, versionsApiError };
 };
 
+export const loadTranslations = async (ref, versions) => {
+    let translationsApiError = false;
```

Comment:
I see that you copied this from the function I wrote above (`loadVersions`). I'm now reading this and realizing how terrible this pattern is. We should not be storing a boolean for whether or not there was an error. We should throw the error and have the outer scope catch it. Happy to huddle to explain.